bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerMetadata.java

```diff
@@ -193,7 +194,8 @@ boolean hasPassword() {
         return hasPassword;
     }
 
-    byte[] getPassword() {
+    @VisibleForTesting
+    public byte[] getPassword() {
```

Comment:
@sijie  I have added "public" here in order to move the test case in another package. I can use reflection if you prefer

Reply:
fine for me.